python/vyos/kea.py

```diff
@@ -328,6 +328,18 @@ def kea_get_leases(inet):
 
     return leases['arguments']['leases']
 
+def kea_delete_lease(inet, ip_address):
+    ctrl_socket = f'/run/kea/dhcp{inet}-ctrl-socket'
```

Comment:
I'm pretty sure the socket path will be used in many more functions in the future than it already is — why not move it to a module-level constant?